mesheryctl/internal/cli/root/system/error.go

```diff
@@ -148,6 +157,42 @@ func ErrFailedGetEphemeralPort(err error) error {
 	)
 }
 
+func ErrProcessingConfig(err error) error {
+	return errors.New(ErrProcessingConfigCode, errors.Alert, []string{"Error processing config"}, []string{"Error processing config", err.Error()}, []string{}, []string{})
+}
+
+func ErrCreatingConfigFile(err error) error {
+	return errors.New(ErrCreatingConfigFileCode, errors.Alert, []string{"Unable to create config file"}, []string{"Unable to create config file"}, []string{}, []string{})
+}
+
+func ErrAddingTokenToConfig(err error) error {
+	return errors.New(ErrAddingTokenToConfigCode, errors.Alert, []string{"Unable to add token to config"}, []string{"Unable to add token to config"}, []string{}, []string{})
+}
+
+func ErrAddingContextToConfig(err error) error {
+	return errors.New(ErrAddingContextToConfigCode, errors.Alert, []string{"Unable to add context to config"}, []string{"Unable to add context to config"}, []string{}, []string{})
```

Comment:
@zakisk will you please see that each error code has a probable cause and a suggested remediation defined?

I realize that some or all of these error codes might not be originally defined by you, but it's time to put our foot down and get on the right side of these, making them truly useful, @abdullah1308.

Reply:
@leecalcote hey, you're right none of these error code are defined by me they were already. and I used them in the same way as they were used by others previously. I should see their usage and suggested remediation :100:

Reply:
@zakisk nice work here , the prs #8208 and #8256 are being worked on for generating error wrappers to be used in different aspect of `mesheryctl`.  When the pr's are merged you can make changes and call the error functions. so we avoid redundancy in writing error functions in the` mesheryctl/pkg/utils/error.go` just as @abdullah1308 Mentioned

**And please sign off your commits**